### PR TITLE
man: add fi_pep_bind to fi_endpoint(3)

### LINKED ISSUE #548: fi_pep_bind missing in fi_endpoint(3)
There are entries for fi_ep_bind and fi_scalable_ep_bind, but no entry for fi_pep_bind. Is this intentional?

### PR DESCRIPTION
**Pull request by bturrubiates**
Add an entry for fi_pep_bind to accompany fi_ep_bind and fi_scalable_ep_bind.

Closes #548 .

### CODE REVIEW

File: man/fi_endpoint.3.md [hunk 1]
```diff
@@ -79,6 +82,8 @@ int fi_ep_bind(struct fid_ep *ep, struct fid *fid, uint64_t flags);
 
 int fi_scalable_ep_bind(struct fid_sep *sep, struct fid *fid, uint64_t flags);
 
+int fi_pep_bind(struct fid_pep *pep, struct fid *fid, unit64_t flags);
```
Comment (goodell) [inline man/fi_endpoint.3.md]: typo: s/unit/uint/

Comment (bturrubiates) [inline man/fi_endpoint.3.md]: Oops. Thanks for noticing. Should I fix in another commit, or revert and fix?

**Comment by sayantansur** [comment]
The signed off by line is missing. Can you amend your commit?

**Comment by bturrubiates** [comment]
Sure, I wasn't sure what I was supposed to do for that.

**Comment by bturrubiates** [comment]
Who do I include in the signed off by line?

**Comment by sayantansur** [comment]
It should have your name and organization. You can look at other commits as an example. Sean can probably point you to the link that describes the IP rules (licensing etc.) for libfabric. I can't seem to find it on the Github site.

**Comment by bturrubiates** [comment]
Is that right?

**Comment by sayantansur** [comment]
Looks right to me.

**Comment by shefty** [comment]
thanks

**Pull request merged**